tests/unit9.test.js

```diff
@@ -0,0 +1,60 @@
+const { test, expect } = require('@playwright/test');
+
+test('Perform Login:', async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+  await page.getByPlaceholder('Username').fill('standard_user');
+  await page.getByPlaceholder('Password').fill('secret_sauce');
+  await page.locator('input[data-test="login-button"]').click();
+  expect(page).toHaveURL('https://www.saucedemo.com/inventory.html');
+
+  expect(page.getByText('Products')).toBeVisible(); 
+  //але мені здається це варік не дуже, бо такий текст 
+  //може ще з часом з*явитись. тут мабуть по data-test краще. чи ні? ось так: 
+  //const title = page.locator('span[data-test="title"]');
+  //expect(title).toBeVisible;
+
+  const shoppingCart =  page.locator('span[data-test="shopping_cart_link"]');
+  expect(shoppingCart).toBeVisible;
+  
+  const products =  page.locator('div[data-test="inventory-item"]');
+  const productCount = await products.count();
+
+  expect(productCount).toBeGreaterThan(1);
+  await page.close();
+  });
+
+test(' Add product to the cart:', async ({ page }) => {
+    await page.goto('https://www.saucedemo.com/');
+    await page.getByPlaceholder('Username').fill('standard_user');
+    await page.getByPlaceholder('Password').fill('secret_sauce');
+    await page.locator('input[data-test="login-button"]').click();
+
+    expect(page).toHaveURL('https://www.saucedemo.com/inventory.html');
```

Comment:
You forget `await` because it async method, please check doc - https://playwright.dev/docs/test-assertions